packages/sdk/src/types/plugins/ModuleFederationPlugin.ts

```diff
@@ -203,6 +203,10 @@ export interface ModuleFederationPluginOptions {
    * Runtime plugin file paths or package name.
    */
   runtimePlugins?: string[];
+  /**
+   * Custom public path function
+   */
+  getPublicPath?: string;
```

Comment:
A user should have a way to specify either `getPublicPath` or `publicPath` as options to the module federation plugin.

Currently in the rspack plugin the `publicPath` is taken from "outside" the module federation config, specifically from `config.output.publicPath`, and is specific only to rspack.

For a bundler like rollup to implement this, `publicPath` has to be present in the module federation config, just like `getPublicPath`.

Maybe find a way to merge both since `getPublicPath` is more powerful than `publicPath`

Reply:
We cannot change the compilers existing public path implementation

Reply:
Just add an option in the module federation config which acts as an override to the compiler's public path.

Reply:
Thats what getPublicPath does? 
__webpack_require__.p = (()=>'thing')()